### PR TITLE
Create accessibility-for-localization.md

### PR DESCRIPTION
**Pull request by 1Copenut**
Capturing technical notes and guidance for teams looking to localize their content. These notes and examples assume Spanish for the moment, but can absolutely be expanded to include other languages as warranted.

### CODE REVIEW

File: platform/accessibility/resources/accessibility-for-localization.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+  ! Screen reader only text must be updated with the visual text to ensure good localization
+  
+  <a href="#">
+    Presentar mi solicitud
+  - <span class="vads-u-visibility--screen-reader"> for education benefits</span>
+  + <span class="vads-u-visibility--screen-reader"> de beneficios educativos</span>
+  </a>
+  ```
+  
+* We'll also want to make sure that `aria-labels` contain full phrases so that the text translates accurately. For example, if a button's visible text is "Enroll now," the aria label text should include "Enroll now in VA health care."
```
Comment (jenstrickland) [inline platform/accessibility/resources/accessibility-for-localization.md:26]: We should not have any buttons that simply say "Enroll now" to meet [WCAG 2.0 Level A - 2.4.4](https://www.wuhcag.com/link-purpose-in-context/), and the aspirational [ WCAG 2.0 Level AAA - 2.4.9,](https://www.wuhcag.com/link-purpose-link-only/).

File: platform/accessibility/resources/accessibility-for-localization.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+* Accessibility experts can provide more guidance for how to make translated content successful for screen readers.
+* We can also wrap individual pieces of translated content in `"lang-es"` so that screen readers will use the right pronunciation to read the translated text. We are going to try this in a test environment for the block of helper text on [va.gov/health-care/how-to-apply](va.gov/health-care/how-to-apply).
```
Comment (jenstrickland) [inline platform/accessibility/resources/accessibility-for-localization.md:28]: The `"lang-es"` may not be correct. Recommend providing the `<span lang="es">Jugador</span>` example.

**Review by jenstrickland — CHANGES_REQUESTED** [review]

File: platform/accessibility/resources/accessibility-for-localization.md (hunk 2 above)
Comment (1Copenut) [reply platform/accessibility/resources/accessibility-for-localization.md:28]: Ack, that one's an oversight. Will correct.

File: platform/accessibility/resources/accessibility-for-localization.md (hunk 1 above)
Comment (1Copenut) [reply platform/accessibility/resources/accessibility-for-localization.md:26]: Updated this text.

**Review by jenstrickland — APPROVED** [review]

**Comment by 1Copenut** [comment]
@peggygannon or @gmaciolek Would either of you have time to review this PR this week? Thank you!

**Pull request merged**